main/test/ca/uwaterloo/flix/language/phase/TestEntryPoints.scala

```diff
@@ -511,4 +511,28 @@ class TestEntryPoints extends AnyFunSuite with TestUtils {
     val result = compile(input, Options.TestWithLibMin)
     expectSuccess(result)
   }
+
+  test("Test.ValidEntryPoint.DefaultHandler.02") {
+    val input =
+      """
+        |pub eff E{
+        |   def op(): Unit
+        |}
+        |mod E {
+        |    @DefaultHandler
+        |    pub def runWithIO(f: Unit -> a \ ef): a \ (ef - E) + IO =
+        |            run {
+        |                f()
+        |            } with handler E {
+        |                def op(k) = {
+        |                    println("Default behaviour");
+        |                    k()
+        |                }
+        |            }
+        |}
+        |def main(): Unit \ E = E.op()
+        |""".stripMargin
+    val result = compile(input, Options.TestWithLibMin)
+    expectSuccess(result)
```

Comment:
We also want a lot more tests:

- One default handler
- Two default handlers
- Three default handlers

Reply:
Checking whether it compiles correctly or not is already a great test. We have lots of those.

For checking "the output", you can just use assertEq.

As for having the IO+effect. Your handler can be pure, and just do if (false) print etc.